book/forms.md

```diff
@@ -501,35 +492,86 @@ def request(url, payload=None):
     # ...
 ```
 
-So---now we've sent the `POST` request. From the point of view of the
-browser, that's about it: the server will respond with a web page and
-the browser will render it in the totally normal way. But to better
-understand the whole cycle---and also to make it easier to test our
-browser's form support---let's take a small detour out of the browser
-and look at how the server will handle these requests.
+So that's how the `POST` request gets sent. Then the server responds
+with an HTML page and the browser will render it in the totally normal
+way. That's basically it for forms on the browser side. But before we
+move on to the next chapter, let me explain how forms are used to
+build browser applications.
+
+::: {.further}
+Something about event dispatching
+:::
 
+How web applications work
+=========================
+
+So... How do forms support web applications? When you use an
+application from your browser---whether you are registering to vote,
+looking at pictures of your baby cousin, or checking your
+email---there are typically[^exceptions] two programs involved: client
+code that runs in the browser, and server code that runs on the
+server. When you click on things or take actions in the application,
+that runs client code, which sends then data to the server via
+HTTP requests.
+
+[^exceptions]: Here's I'm talking in general terms. There are some
+    browser applications without a server, and others where the client
+    code is exceptionally simple and almost all the code is on the
+    server.
+
+For example, imagine a simple message board application. The server
+stores the state of the message board---who has posted what---and has
+logic for updating that state. But all the actual interaction with the
+page---drawing the posts, letting the user enter new ones---happens in
+the browser. Both components are necessary.
+
+The browser and the server interact over HTTP. The browser first makes
+a `GET` request to the server to load the current message board. You
+can think of this as downloading and running the client code in the
+browser. The user interacts with the browser to type a new post, and
+submits it to the server (say, via a form). That causes the browser to
+make a `POST` request to the server, which instructs the server to
+update the message board state. The response to this request is a new
+HTML page with the new state of the message board, taking into account
+the requested changes.
+
+Forms are a simple, minimal introduction to this cycle of request and
+response and make a good introduction to how browser applications
+work. They're also implemented in every browser and have been around
+for decades. These days web applications still use the form elements,
```

Comment:
```suggestion
for decades. These days, however, web applications still use the form elements,
```

Reply:
I wanted to frame this postively (continuity to modern web applications) not negatively (things have changed). To reinforce that, I added a "nonetheless later"